src/components/directives/scaleToFit.js

```diff
@@ -0,0 +1,48 @@
+var scaleToFit = angular.module("bawApp.directives.scaleToFit", []);
+
+
+scaleToFit.directive("scaleToFit",
```

Comment:
Perhaps leave a comment describing the function of this directive. I'm not sure of it's intended use.

Reply:
I have removed this directive. It was originally done for the cases of example annotations that are too wide for the browser window, but we have decided to simplify things by making it the responsibility of the CS project curator to only select reasonable example annotations.